### PR TITLE
test: add contract interaction verify attestation test

### PR DESCRIPTION
**Pull request by SweeXordious**
## Checklist

- [ ] New and updated code has appropriate documentation
- [ ] New and updated code has new and/or updated testing
- [ ] Required CI checks are passing
- [ ] Visual proof for any user facing features like CLI or documentation updates
- [ ] Linked issues closed with keywords

### CODE REVIEW

File: x/qgb/orchestrator/test/contract_interaction_test.go [hunk 1]
```diff
@@ -204,6 +208,108 @@ func (s *QGBTestSuite) TestUpdateValset() {
...
+	for i := 0; i < count; i++ {
+		heights[i] = uint64(i)
+		root := bytes.Repeat([]byte{byte(i)}, 32)
+		roots[i] = root
+
+		tuple := wrapper.DataRootTuple{
+			Height:   big.NewInt(int64(i)),
+			DataRoot: *(*[32]byte)(root),
+		}
+		encodedTuple, err := rpccore.EncodeDataRootTuple(uint64(i), tuple.DataRoot)
```
[the remark this answers is not in the document]
Comment (SweeXordious) [reply x/qgb/orchestrator/test/contract_interaction_test.go:304]: it is in the latest release

**Comment by evan-forbes** [comment]
what's the status of this PR? apologies for missing this review, I'll dig in first thing tmrw

**Comment by SweeXordious** [comment]
Thanks. It's an optional PR, it just adds a test to the qgb-integration branch that we're deprecating. So, we can close it without merging or merge it without the CI passing, doesn't matter in my opinion

**Comment by evan-forbes** [comment]
ahh I see, ok sounds good. lets merge just to save it

**Pull request merged**